_docs/reference/cli/cluster.md

```diff
@@ -5,4 +5,53 @@ anchor: reference
 module: reference/cli/cluster
 ---
 
-{% include wip.md %}
+# Cluster
```

Comment:
Yeah, it's not linked in the side menu so not easy to find unless you look in Github.